source/documentation/administration_guide/topics/Setting_Load_and_Power_Mgmt_Policies_for_Hosts.adoc

```diff
@@ -20,6 +20,12 @@ The *evenly_distributed* and *power_saving* scheduling policies allow you to spe
 .. Set the time (in minutes) that a host can run a CPU load outside of the defined utilization values before the scheduling policy takes action in the *CpuOverCommitDurationMinutes* field.
 .. Enter the CPU utilization percentage at which virtual machines start migrating to other hosts in the *HighUtilization* field.
 .. Optionally, in the *HeSparesCount* field, enter the number of additional self-hosted engine nodes on which to reserve enough free memory to start the {engine-name} virtual machine if it migrates or shuts down. See xref:Configuring_Memory_Slots_Reserved_for_the_SHE[Configuring Memory Slots Reserved for the self-hosted engine] for more information.
+.. Optionally, to prevent the host from over utilization of all the physical CPUs, define the virtual CPU to physical CPU ratio - *VCputophysicalCpuRatio* with a value between 0.1 and 2.5. When this parameter is set, hosts with a lower CPU utilization are preferred when scheduling a virtual machine.
++
+If adding an additional virtual machine causes the ratio to exceed 2.5, both
+the *VCputophysicalCpuRatio* and the CPU utilization are considered.
++
+In a running environment, if the host *VCputophysicalCpuRatio* exceeds 2.5, some virtual machines may be load balanced and moved to hosts with a lower *VCputophysicalCpuRatio*.
```

Comment:
"may" is ambiguous -- I suggest you use either "might" or "can," depending on the context.  Given that only a user with the correct permissions ia allowed to execute a given command, there is no difference between "may" and "can."